runs when getcap is available, and setting a cap in fakeroot is forbidden so it fails
--
--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -13,8 +13,7 @@ TESTS =						\
   t.perlstat					\
   t.tar						\
   t.touchinstall				\
-  t.truereturn					\
-  t.xattr
+  t.truereturn
 
 suffix =
 TESTS_ENVIRONMENT =				\
